docs/user/commercial/index.rst

```diff
@@ -15,24 +46,25 @@ Private repositories and private documentation
 Additional build resources
     Do you have a complicated build process that uses large amounts
     of CPU, memory, disk, or networking resources?
-    Our commercial offering has much higher default resources
-    that result in faster documentation build times
-    and we can increase it further for very demanding projects.
+    Commercial subscriptions offer more resources
+    which results in faster documentation build times.
+    Resources can be increased further for very demanding projects.
```

Comment:
```suggestion
    Commercial subscriptions offer more resources
    which results in faster documentation build times.
    We can also customize builders, such as with a GPU or multiple CPUs.
```

Reply:
Super! I really didn't want to remove it, but I also didn't like the fact that the previous sentence kind of implied that we are running things artificially slowly.. like an Intel Celeron CPU sold with disabled circuits :)

Reply:
More clarity FTW!

Reply:
It's a win/win, an upsell and more explicit :)